modules/op-about-tasks.adoc

```diff
@@ -5,9 +5,9 @@
 [id="about-tasks_{context}"]
 = Tasks
 
-_Tasks_ are the building blocks of a Pipeline and consist of sequentially executed Steps. Tasks are reusable and can be used in multiple Pipelines.
+_Tasks_ are the building blocks of a Pipeline and consists of sequentially executed steps. It is essentially a function of inputs and outputs that form the Pipeline build. A Task can run individually or as a part of the pipeline. Tasks are reusable and can be used in multiple Pipelines. 
```

Comment:
1. Tasks and pipelines are sometimes capitalized, sometimes not, shouldn't we unify that?
2. `that form the Pipeline build`... I'm not sure what you want to say here

Reply:
We will be creating another PR to tackle all the capitalization of the CRD, post the release rush. We will make all of them small case as per the OCP docs guidance.

Reply:
@HarshCasper we can remove this sentence: It is essentially a function of inputs and outputs that form the Pipeline build.
The task run part explains this in a better manner.

Reply:
Done!